internal/testutil/pkgdata.go

```diff
@@ -265,3 +265,11 @@ func MakeDeb(entries []TarEntry) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+func MakeTestDeb(entries []TarEntry) []byte {
+	data, err := MakeDeb(entries)
```

Comment:
Looks fine, but can we please just name this function as `MustMakeDeb`?  There's a well established convention of using the `Must` prefix in exactly these cases.

Reply:
Thanks. That's definitely a better name. Updated.